app/assets/styles/app.scss

```diff
@@ -180,7 +180,7 @@ h1.ui.header, h2.ui.header {
 }
 
 // Message
-.ui.positive.message {
+.ui.positive.message, .ui.success.message, .ui.error.message {
```

Comment:
How is this? The success is the teal and the error is the app secondary color

<img width="742" alt="screen shot 2017-02-14 at 3 30 08 pm" src="https://cloud.githubusercontent.com/assets/5404536/22947882/88089354-f2ca-11e6-8c85-e281c9ec7936.png">
<img width="723" alt="screen shot 2017-02-14 at 3 30 23 pm" src="https://cloud.githubusercontent.com/assets/5404536/22947881/8807749c-f2ca-11e6-8a0e-77ebd0e217dc.png">